packages/webapp/tailwind.config.js

```diff
@@ -31,17 +31,22 @@ module.exports = {
                 'dark-700': '#18181B',
                 'dark-800': '#09090B',
                 'bg-dark-blue': '#182633',
-                'state-green-900': '#84D65A',
-                'state-green-400': '#84D65A',
                 'row-hover': '#0d0d14',
-                white: '#FFFFFF'
+                white: '#FFFFFF',
+
+                // From Figma
+                'green-base': '#84D65A'
             },
             width: {
                 largebox: '1200px',
                 largecell: '480px'
             },
             fontSize: {
+                s: '13px',
```

Comment:
14px is also used a lot, seems a bit strange for things to go to 13px. Something to confirm with the design I suppose

Reply:
Yes, we increased the font-size but I think 14 was too big. You can find the figma in `Product > Logs V2`